tests/test_latest_pointer.py

```
@@ -0,0 +1,138 @@
+    tmp_path, symlinks_supported
+):
+    if not symlinks_supported:
+        pytest.skip("os.symlink not supported on this platform")
+    target = tmp_path / "v2.7.0"
+    target.mkdir()
+
+    with patch("os.symlink", wraps=os.symlink) as mock_symlink:
+        update_latest_pointer(tmp_path, target.name)
+
+    mock_symlink.assert_called()
+    args, kwargs = mock_symlink.call_args
```

Ruff (0.15.12)

[warning] Unpacked variable `args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)